native: `ncache add` shell command results in segfault

Found a way to cause a segfault in 'native': run the default example and simply call `ncache add` without any further argument -> segfault:
```
RIOT native interrupts/signals initialized.
LED_RED_OFF
LED_GREEN_ON
RIOT native board initialized.
RIOT native hardware initialization complete.

main(): This is RIOT! (Version: )
RIOT network stack example application
All up, running the shell now
> ncache add
ncache add
make: *** [term] Segmentation fault
```

I don't have time to look into this right now, but maybe we should fix this?! :-)

--- a/sys/shell/commands/sc_ipv6_nc.c
+++ b/sys/shell/commands/sc_ipv6_nc.c
@@ -237,10 +237,10 @@ int _ipv6_nc_manage(int argc, char **argv)
     }
 
     if (argc > 1) {
-        if (strcmp("add", argv[1]) == 0) {
+        if ((argc > 2) && (strcmp("add", argv[1]) == 0)) {
             return _ipv6_nc_add(argc - 2, &argv[2]);
         }
-        if (strcmp("del", argv[1]) == 0) {
+        if ((argc > 2) && (strcmp("del", argv[1]) == 0)) {
             return _ipv6_nc_del(argv[2]);
         }
         if (strcmp("reset", argv[1]) == 0) {

